src/GovUk.Education.ExploreEducationStatistics.Content.Services.Tests/Cache/MethodologyCacheServiceTests.cs

```diff
@@ -192,7 +192,8 @@ public async Task GetSummariesByPublication_PublicationNotFound()
 
         VerifyAllMocks(PublicBlobCacheService);
 
-        result.AssertRight([]);
+        var viewModels = result.AssertRight();
+        Assert.Empty(viewModels);
```

Comment:
I had to make the same change on my branch! It did compile but Rider didn't like it for whatever reason. I'm guessing dotnet version mismatch or something

Reply:
Haha yeah!